runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcIndicatorsConfig.java

```diff
@@ -99,12 +84,41 @@ default boolean highlightMenuNames()
 	}
 
 	@ConfigItem(
-		position = 6,
 		keyName = "showRespawnTimer",
 		name = "Show respawn timer",
 		description = "Show respawn timer of tagged NPCs")
 	default boolean showRespawnTimer()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "highlightStyleTile",
+		name = "Highlight Style Tile",
```

Comment:
These config names don't really make sense as-is. Something like this for each of the added configs would read much better.
```suggestion
		name = "Highlight NPC Tiles",
```